prettier/tests/run.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: prettier-pipeline
+spec:
+  workspaces:
+  - name: shared-workspace
+    # description: The git repo will be cloned into this workspace and the config-defaults.yaml will be read from it.
+  tasks:
+  - name: fetch-repo
+    taskRef:
+      name: git-clone
+    workspaces:
+    - name: output
+      workspace: shared-workspace
+    params:
+    - name: url
+      value: https://github.com/tektoncd/pipeline.git
```

Comment:
I haven't checked how long this run, but should we maybe use a repository that's not as big as `pipeline`? The current catalog tests can get a long time to run and using a smaller git repository may help, i.e: something like a helloworld https://github.com/go-training/helloworld repository?

Reply:
I would prefer to not depend on an external repo. If/when they make a change to their repo, it might break our tests downstream.

Since it looks like the catalog repo is available for tests anyway, how about we run prettier over the catalog? It has an added benefit that we'll now have a catalog tasks to ensure consistent formatting! 😄

Reply:
That makes sense. Should I use the commit SHA from the `master` branch or `v1beta1` branch or both okay?

Reply:
I think we may not even need the git-clone task.
Based on https://github.com/tektoncd/catalog/pull/351#issuecomment-642575792 it sounds like the catalog repo is already available to running tests by default? So we may just be able to run the linter on the local path.

You'd have to play around with the test environment to find out for sure.